### DISCUSSION TITLE
MRF Validator incorrect results for NDC Code Type

### DISCUSSION THREAD

**Discussion by BBDURMC** [CMSgov/price-transparency-guide#739] [Q&A] [answered]
The MRF validator is not coded correctly for line items with NDC Code Type.

An error is produced which reads: "If code type is NDC, then the corresponding drug unit of measure and drug type of measure data element must be encoded."
These measurements/data elements are not required until 1/1/25.
Please correct the validator so that is does not fail on this combination of events. 
The NDC code Type should be valid for 7/1/24 without the corresponding measures.

If it is not, then please indicate the correct Code Type to use for NDC when a facility will not be providing units of measure until 1/1/25. Should Other be used?

Thank you!
Beverly Dollinger
beverly_dollinger@urmc.rochester.edu

**Comment by shaselton-usds** [comment, marked as answer]
Hi @BBDURMC 

This Github repository is for the Transparency in Coverage regulation. I believe you referring to OPPS/ASC rule that is for the Hospital Price Transparency.

Where you pulled that text can be found in this repo: https://github.com/CMSgov/hospital-price-transparency
The validator tools for the Hospital Price Transparency can be found here: https://github.com/CMSgov/hpt-tool
If you wish to use the Hospital Price Transparency validator online directly, you can find that here: https://cmsgov.github.io/hpt-tool/
